apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Renew.purs

```diff
@@ -0,0 +1,329 @@
+module MittKonto.Main.UserView.Subscription.Renew where
+
+import MittKonto.Main.UserView.Subscription.Types as Types
+
+import Prelude
+
+import Bottega (BottegaError(..))
+import Bottega.Models (FailReason(..), OrderState(..), PaymentMethod(..), PaymentTerminalUrl)
+import Bottega.Poller as Poller
+import Control.Monad.Except.Trans (ExceptT(..), runExceptT)
+import Data.Array as Array
+import Data.Either (Either(..))
+import Data.Foldable (foldMap)
+import Data.Maybe (Maybe(..), fromMaybe, maybe, isJust, isNothing)
+import Effect (Effect)
+import Effect.Aff as Aff
+import Effect.Class (liftEffect)
+import Effect.Class.Console as Console
+import KSF.AsyncWrapper as AsyncWrapper
+import KSF.Api.Subscription as Subscription
+import KSF.Api.Subscription (toString) as Subsno
+import KSF.Grid as Grid
+import KSF.Helpers as Helpers
+import KSF.InputField (inputLabel)
+import KSF.InputField.Checkbox as Checkbox
+import KSF.User as User
+import MittKonto.Main.UserView.Subscription.Types (RenewSubscription)
+import MittKonto.Wrappers.Elements as WrapperElements
+import React.Basic (JSX)
+import React.Basic.Hooks (Component, component, useEffectOnce, useState, (/\))
+import React.Basic.Hooks as React
+import React.Basic.DOM as DOM
+import React.Basic.DOM.Events (preventDefault, targetValue)
+import React.Basic.Events (handler, handler_)
+
+type Form =
+  { months :: Int
+  , cents :: Int
+  , shownPrice :: Int
+  , paymentMethod :: PaymentMethod
+  , confirm :: Maybe Boolean
```

Comment:
Is this step necessary? I mean, adding one more step after clicking "Köp nu"

Reply:
I asked about it on Slack and the consensus appeared to be that it should have that extra step.